src/plextrac

```diff
@@ -271,20 +271,31 @@ function mod_start() {
   # Enable database migrations on startup
   compose_client --profile=database-migrations up -d --remove-orphans
 
-  info "Waiting for application startup"
+  waitTimeout=${1:-60}
+  info "Waiting up to ${waitTimeout}s for application startup"
   local progressBar
   # todo: extract this to function waitForCondition
   # it should take an optional param which is a function
   # that should return 0 when ready
   (
-    for i in `seq 1 30`; do
+    while true; do
```

Comment:
I really hate having this be unbounded, BUT so far in my testing it has not managed to avoid getting killed.